array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map((num) => num * 2);
+console.log('Doubled:', doubled);
+
+const prices = numbers.map((num) => '$' + num.toString() + '.00');
```

Comment:
The best way to convert to a price is:
`` `$${num.toFixed(2)}` ``

Your approach only works by chance because all the numbers are integers.